launch/Mako_G-319.launch

```diff
@@ -18,6 +18,6 @@
     <arg name="pixel_format" value="BayerRG8"/>
     <arg name="width" value="2064"/>
     <arg name="height" value="1544"/>
-    <arg name="stream_bytes_per_second" value="119824282"/>
+    <arg name="stream_bytes_per_second" value="124000000"/>
```

Comment:
Why the change from a seemingly random number to 1 million bytes/sec less than the theoretical max of GigE? Should it be 125 * 10^6?

Reply:
124M is the maximum for the Mako G-319 according to their docs. See here page 51:
https://cdn.alliedvision.com/fileadmin/content/documents/products/cameras/Mako/techman/Mako_TechMan_en.pdf
Keep in mind this launch file is specific to the Mako G-319 model.